src/components/DropMenu/DropMenu.jsx

```diff
@@ -429,6 +451,7 @@ const DropMenu = createClass({
 						'&-Option-is-selected': isSelected,
 						'&-Option-is-disabled': isDisabled,
 						'&-Option-is-null': _.isNull(optionIndex),
+						'&-Option-is-wrapped': isWrapped,
```

Comment:
I'm getting confused a bit by this prop name -- it's called `isWrapped` but it actually applies `nowrap` if it is `true` and then the example for using it is called `no-wrapping`. Is that correct?